docs/core/porting/breaking-changes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Breaking changes
+description: Breaking changes between releases of .NET and how it impacts compatibility.
+author: stebon
+ms.date: 06/14/2021
+---
+# Breaking Changes
+
+Changes that affect compatibility, otherwise known as breaking changes, will occur between versions of .NET. This is particularly impactful when porting from .NET Framework to .NET due to certain technologies not being available as well as the changes needed to make .NET a cross-platform framework.
+
+We strive to maintain a high level of compatibility between .NET versions, so while breaking changes will occur, they are carefully considered.
+
+Before upgrading major versions check the breaking changes documentation for changes that might impact you.
```

Comment:
There should be commas after the intro clauses. Look for these in other sentences too.
```suggestion
Before upgrading major versions, check the breaking changes documentation for changes that might impact you.
```